libbeat/parser/parser.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package filestream
+package parser
```

Comment:
I wonder about the package name. Maybe move it to `libbeat/common/parser` ?

Reply:
Moved it to `libbeat/reader/parser`.